test/UserTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Icepace\User;
+use PHPUnit\Framework\TestCase;
+require_once 'vendor/autoload.php';
```

Comment:
`require_once` should be at the top of the file above `use`

Reply:
moved

Reply:
also at the very top you need:
```php
namespace Icepace\Test;
```

also the vendor line needs to say `require ../vendor/autoload.php`;

Reply:
fixed